lib/testutils/src/git.rs

```diff
@@ -56,23 +56,24 @@ pub fn init_bare(directory: impl AsRef<Path>) -> gix::Repository {
     .to_thread_local()
 }
 
-pub fn clone(dest_path: &Path, url: &str) -> gix::Repository {
-    let mut prepare_fetch = gix::clone::PrepareFetch::new(
-        url,
-        dest_path,
-        gix::create::Kind::WithWorktree,
-        gix::create::Options::default(),
-        open_options(),
-    )
-    .unwrap();
-    let (mut prepare_checkout, _outcome) = prepare_fetch
-        .fetch_then_checkout(gix::progress::Discard, &gix::interrupt::IS_INTERRUPTED)
-        .unwrap();
-    let (repo, _outcome) = prepare_checkout
-        .main_worktree(gix::progress::Discard, &gix::interrupt::IS_INTERRUPTED)
-        .unwrap();
+pub fn clone(dest_path: &Path, repo_url: &str) -> gix::Repository {
+    // gitoxide doesn't seem to write the `refs/remotes` to disk on clone
+    // (https://github.com/GitoxideLabs/gitoxide/issues/1865)
+    //
+    // it'd be nice not to have the hard-dependency on `git` at test time, but it
+    // seems unavoidable
+    assert!(std::process::Command::new("git")
+        .args(["clone", repo_url, dest_path.to_str().unwrap()])
+        .stdin(std::process::Stdio::null())
+        .stdout(std::process::Stdio::null())
+        .stderr(std::process::Stdio::null())
```

Comment:
nit: might be better to not redirect stdout and stderr so we can debug problem on error.

Another option is to use `.output()` and include the output in assertion message.
https://doc.rust-lang.org/stable/std/process/struct.Command.html#method.output